libs/types/src/election_utils.ts

```diff
@@ -533,6 +533,21 @@ function preprocessElection(value: unknown): unknown {
     };
   }
 
+  // Add sheetNumber to grid positions
+  /* istanbul ignore next */
+  if (election.gridLayouts) {
+    election = {
+      ...election,
+      gridLayouts: election.gridLayouts.map((gridLayout) => ({
+        ...gridLayout,
+        gridPositions: gridLayout.gridPositions.map((gridPosition) => ({
+          ...gridPosition,
+          sheetNumber: gridPosition.sheetNumber ?? 1,
+        })),
+      })),
+    };
+  }
```

Comment:
Great question. I thought about doing that.

I've run into a few situations where the election parsing isn't backwards compatible when we're working with a real backup, and it's an annoying roadblock. I've found it's harder to figure out what to do in those situations since the context on what changed isn't fresh. So I think of adding the backwards compatibility as a gift to my future self.

The approach I've taken for the last few changes is:
- Update all of the fixtures _without_ the backwards compatibility
- Then add the backwards compatibility as a last step

I slightly broke this approach here by leaving the m17-backup fixture, mainly because it's hard to update the election definition that's stored in the db file in the backup.

Reply:
> I've found it's harder to figure out what to do in those situations since the context on what changed isn't fresh. So I think of adding the backwards compatibility as a gift to my future self.

Good point!

So long as we keep backwards compatibility logic consolidated and clearly labeled, it seems reasonable to add it when easy

Reply:
Sounds good! I've been following the pattern of putting it in this `preprocessElection` function, which seems to be working well